framework/framework.go

```diff
@@ -25,7 +27,9 @@ import (
 type Config struct {
 	// Dependencies.
 
-	Informer informer.Interface
+	CRD       *crd.CRD
+	CRDClient *crdclient.CRDClient
```

Comment:
Note my other PR where this got moved. Please see https://github.com/giantswarm/operatorkit/pull/124 first so we can merge and align here.